packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -235,13 +240,39 @@ const SchedulerForm: FC<Props> = ({
     isThresholdAlert,
     itemsMap,
 }) => {
+    const isDashboard = resource && resource.type === 'dashboard';
+    const isDashboardTabsEnabled = useFeatureFlagEnabled(
+        FeatureFlags.DashboardTabs,
+    );
+    const { data: dashboard } = useDashboardQuery(resource?.uuid, {
+        enabled: isDashboard,
+    });
+
+    const isDashboardTabsAvailable =
+        dashboard?.tabs !== undefined && dashboard.tabs.length > 0;
+
     const form = useForm({
         initialValues:
             savedSchedulerData !== undefined
-                ? getFormValuesFromScheduler(savedSchedulerData)
+                ? getFormValuesFromScheduler({
+                      ...savedSchedulerData,
+                      ...(isDashboardScheduler(savedSchedulerData) && {
+                          selectedTabs: isDashboardTabsAvailable
```

Comment:
we can probably extract all this logic in a variable outside the method call, just for readability.